### ISSUE TITLE
[自荐] 面向开发阅读国内外技术文章的 App

### ISSUE THREAD

**Issue by wxyudl** [ruanyf/weekly#1007]
离阮大这么近，好紧张。

这个是一款手机应用名字叫【**极客阅读**】，是我业余时间做了。功能很纯粹就是看国内外的技术文章 （非保存、转载）。

会自动收集从国内、国外的技术博客、公众号文章。然后推送手机端（每小时更新一次），这样就能**一站式的阅读到互联网技术大咖们的文章**了 。

**应用截图：**

![image](https://user-images.githubusercontent.com/9672031/70771775-515fc500-1dad-11ea-9b92-f24f733bb69e.png)

---

**下载体验：**

感兴趣的小伙伴，[点击这里了解和体验 App](https://www.yuque.com/docs/share/74c8695f-d2ce-4dbd-aee1-b92be37ecdf4)（个人开发的，应用市场搜不到。精力有限**目前只有安卓版**）

**Comment by re4388** [comment]
It would be lovely to have a web app or other convenient form to see how this works. People like me who don't have Android phone can't enjoy this.

**Comment by wxyudl** [comment]
@re4388 Yes. Web version is under development, the whole tech stack is simple.

App: React-Native
Back-end: NestJS
DB: MongoDB
Crawler: Scrapy

:-)

**Comment by re4388** [comment]
> @re4388 Yes. Web version is under development.
> :-)

Good to know!

**Comment by wxyudl** [comment]
Web 版的在这： http://geeker-read.com/

还有一些功能在完善中... 嘻嘻

<img width="1302" alt="image" src="https://user-images.githubusercontent.com/9672031/71308783-aa79c980-243b-11ea-8338-e7a38fbab71b.png">
